integration/common/tests/great_expectations/test_great_expectations_validation_action.py

```diff
@@ -37,7 +37,15 @@
     ExpectationSuiteIdentifier,
     ValidationResultIdentifier,
 )
-from great_expectations.dataset import PandasDataset, SqlAlchemyDataset
+
+try:
```

Comment:
Do you need this `try/except` block? I looked at `0.15.35` and `0.15.50`, the last `0.15.X` release and importing from `great_expectations.dataset` should work.